tests/worktree/bare.c

```diff
@@ -50,3 +50,13 @@ void test_worktree_bare__add(void)
 	git_worktree_free(wt);
 	git_repository_free(wtrepo);
 }
+
+void test_worktree_bare__added_worktree_validates(void)
+{
+	git_worktree *wt;
+
+	cl_git_pass(git_worktree_add(&wt, g_repo, "name", WORKTREE_REPO, NULL));
+	cl_assert_equal_i(0, git_worktree_validate(wt));
```

Comment:
I would've just added this additional call to the previous test, but yeah, doesn't really matter too much.

Reply:
Also it should probably be `cl_git_pass` instead of `cl_assert_equal_i(0, ...`

Reply:
Fixed